packages/client/src/components/AppNavBar/AppNavBar.tsx

```diff
@@ -61,11 +86,33 @@ const pageToButton = (page: typeof PAGES[number]) => (
     {page.name}
   </Button>
 );
+
+const menuItems = (
+  pages: typeof PAGES | typeof LEFT_PAGES | typeof RIGHT_PAGES | typeof USER_PAGES | typeof PARTNER_PAGES,
+) => {
```

Comment:
```suggestion
const menuItems = (pages: readonly { name: string; pathname: string }[]) => {
```
Może taki typ wystarczy jeśli nie korzystasz z kokretnych literałów? `{ name: string; pathname: string }` + `color` można nawet wyciągnąć jako osobny `type`def i skorzystać z niego też w `pageToButton`.